app/CPReflectionService/CPReflectionService.m

```diff
@@ -61,4 +61,14 @@ - (void)allPods:(void (^ _Nonnull)(NSArray<NSString *> * _Nullable pods, NSError
   }];
 }
 
+- (void)updateSpecRepos:(void (^)(NSError * _Nullable))reply
+{
+  [RBObject performBlock:^{
+    [RBObjectFromString(@"Pod::App") update_spec_repos];
```

Comment:
@alloy assuming this takes a while ( cause of `git` / `github.com` ) would this block the ruby thread, e.g. making it not possible to do the parsing for a podfile?